src/tmt300rs/hid-tmt300rs.c

```diff
@@ -501,17 +529,23 @@ int t300rs_stop_effect(void *data, struct tmff2_effect_state *state)
 }
 
 static void t300rs_fill_envelope(struct t300rs_packet_envelope *packet_envelope,
-		int16_t level, uint16_t duration, struct ff_envelope *envelope)
+		struct ff_envelope *envelope)
+{
+	// Note: minimal length limitations are not enforced
+	// as per my testing, the wheel can handle lower values well
```

Comment:
Nit: I also tend to leave comments of the form "my testing" or "I'm doing something here" but you might want to either differentiate yourself of use passive voice here

Reply:
Is that ok?
"Note: Minimal length limitations are not enforced, as testing shows that the wheel can handle lower values well"

Reply:
Sounds good.